index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
+  <!-- title  -->
+  <title>Talent Land Mexico 2023</title>
+  <link rel="icon" type="image/x-icon" href="./assets/favicon.png">
+  <!-- CSS Style -->
+  <link rel="stylesheet" href="./css/style.css">
+  <!-- Google Font -->
+  <script src="https://kit.fontawesome.com/e52d74ec85.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Rubik+Dirt&display=swap" rel="stylesheet">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div>
+      <ul>
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <div>
+      <img src="./assets/logo-talent-land.png" alt="">
+      <span id="openOnClick"><a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776</a></span>
+      <nav>
+        <ul>
+          <li><a href="./about.html"> About</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </nav>
+      <div class="dropdown">
+        <ul id="options" class="dropdown-content">
+          <li><a href="./about.html"> About</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </div>
+      <div>
+        <p>CC Campaing</p>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="headline">
+      <div class="container">
+        <div class="row">
+          <p>"Hello!! Changing the world with talent"</p>
+        </div>
+        <div>
+          <h1>Talent Land Mexico 2023</h1>
+        </div>
+        <div>
+          <p>Talent Land 2023 will be dedicated to talent development, so that the participants can meet with experts
+            from different locations who will share their experience and knowledge.</p>
+        </div>
+        <div>
+          <p>2023.04.01(Sat) ~ 04(Tue)</p>
+        </div>
+        <div>
+          <p>Expo Guadalajara. Av. Mariano Otero 1499, Verde Valle, 44550 Guadalajara, Jal. Mexico.</p>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] It is required to personalize the content of your page so as to make your pages unique. However, it is equally required to follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) which includes:
 -  Colors.
 -  Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
 -  Layout: composition and space between elements. 
 At this moment, the color used on the original design is different from the color your design carries. Kindly follow the design guidelines in that regard.

### Your Design
![your-design](https://user-images.githubusercontent.com/26597432/233477290-81ab7d1b-fe8a-4509-be6a-d8280847a51b.png)

### Guiding Template
![temp](https://user-images.githubusercontent.com/26597432/233477335-f786500b-f428-446f-b097-e56d604138bc.png)